tracing/span.go

```diff
@@ -161,7 +161,9 @@ func (s Span) StopTime() time.Time {
 // This uses the the logger provided by the underlying tracing.Tracer used to
 // publish the Span.
 func (s Span) logError(ctx context.Context, msg string, err error) {
-	s.trace.tracer.logger.Log(ctx, msg+err.Error())
+	if tracer := s.trace.internalTracer(); tracer != nil {
+		tracer.logger.Log(ctx, msg+err.Error())
+	}
```

Comment:
in the interest of fallbacks, should this also do something if internalTracer isn't found?  like fall back to the global logger?

Why does this inject its own logger anyway? 🤔